src/lib/server/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { constants } from '@clerk/backend/internal'
+import { env as privateEnv } from '$env/dynamic/private'
```

Comment:
Is there a reason for these being imported with "dynamic" instead of "static"?

Reply:
I might have a [solution](https://github.com/wobsoriano/svelte-clerk/blob/89ccf5e89d145b33259709588e11fc1edefd2dfc/packages/svelte-clerk/src/lib/server/constants.ts#L6) for this